### PR TITLE
localize book models & views && fix mistake in add_book js

### PR DESCRIPTION
**Pull request by fadyyousry**

### CODE REVIEW

File: app/views/authors/index.html.erb [hunk 1]
```diff
@@ -12,8 +12,8 @@
     <% @authors.each do |author| %>
       <tr>
         <td><%= author.name %></td>
-        <td><%= link_to 'Show', author %></td>
-        <td><%= link_to 'Destroy', author, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <td><%= link_to t(:show, scope: [:default]), author %></td>
+        <td><%= link_to t(:destroy, scope: [:default]), author, method: :delete, data: { confirm: 'Are you sure?' } %></td>
```
Comment (Magdyx) [inline app/views/authors/index.html.erb:16]: you forgot the confirm message

File: app/views/books/_addable_field.html.erb [hunk 2]
```diff
@@ -13,4 +13,4 @@
         </div>
       <% end %>
 <% end %>
-<%= button_tag "New #{singular.capitalize}", :type => 'button', id: "add_#{singular}"%>
\ No newline at end of file
+<%= button_tag button_name, :type => 'button', id: "add_#{singular}"%>
```
Comment (Magdyx) [inline app/views/books/_addable_field.html.erb:16]: I like the idea of this partial a lot. 
What is `button_name`

File: app/views/books/index.html.erb [hunk 3]
```diff
@@ -32,14 +32,14 @@
         <td><%= book.is_pdf %></td>
         <td><%= book.quantity %></td>
         <td><%= book.price %></td>
-        <td><%= link_to 'Show', book %></td>
-        <td><%= link_to 'Edit', edit_book_path(book) %></td>
-        <td><%= link_to 'Destroy', book, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <td><%= link_to t(:show, scope: [:default]), book %></td>
+        <td><%= link_to t(:edit, scope: [:default]), edit_book_path(book) %></td>
+        <td><%= link_to t(:destroy, scope: [:default]), book, method: :delete, data: { confirm: 'Are you sure?' } %></td>
```
Comment (Magdyx) [inline app/views/books/index.html.erb:37]: localization in the confirm message is missing

File: config/locales/en.yml [hunk 4]
```diff
@@ -22,4 +28,23 @@ en:
       email: "Enter Your Email"
       password: "Enter Your Password"
       confirm_password: "Confirm Your Password"
+  book:
+    massages:
```
Comment (Magdyx) [inline config/locales/en.yml:32]: typo

File: config/locales/en.yml [hunk 5]
```diff
@@ -22,4 +28,23 @@ en:
...
+      successful_update: "Book was successfully updated."
+      successful_destroy:  "Book was successfully destroyed."
+    view:
+      add_book: "Add Book"
+      new_book: "New Book"
+      create_book: "Create Book"
+      new_categorey: "New Category"
+      new_author: "New Author"
+  author:
+    massages:
```
Comment (Magdyx) [inline config/locales/en.yml:43]: typo

File: config/locales/en.yml [hunk 6]
```diff
@@ -22,4 +28,23 @@ en:
...
+      successful_destroy:  "Author was successfully destroyed."
+
+  category:
+    massages:
```
Comment (Magdyx) [inline config/locales/en.yml:47]: typo

File: app/views/books/_addable_field.html.erb (hunk 2 above)
Comment (fadyyousry) [reply app/views/books/_addable_field.html.erb:16]: New Author or New Category

File: app/views/books/index.html.erb (hunk 3 above)
Comment (fadyyousry) [reply app/views/books/index.html.erb:37]: yes I forgot it

File: app/views/authors/index.html.erb (hunk 1 above)
Comment (fadyyousry) [reply app/views/authors/index.html.erb:16]: yes

**Pull request merged**